entgql/template.go

```diff
@@ -83,3 +82,8 @@ func filterNodes(nodes []*gen.Type) ([]*gen.Type, error) {
 	}
 	return filteredNodes, nil
 }
+
+// TemplateFuncs contains the extra template functions used by entgql
+var TemplateFuncs = template.FuncMap{
+	"filterNodes": FilterNodes,
+}
```

Comment:
```suggestion
// TemplateFuncs contains the extra template functions used by entgql.
var TemplateFuncs = template.FuncMap{
	"filterNodes": FilterNodes,
}
```

Looks good. Please move it close to the template variables above. Thanks 🙏